Gopkg.lock

```diff
@@ -104,6 +104,12 @@
   revision = "f3f9494671f93fcff853e3c6e9e948b3eb71e590"
 
 [[projects]]
+  name = "github.com/go-stack/stack"
```

Comment:
AFAIU this file is generated by the package manager, rather than written by hand.